Apps/Playground/windows/Playground/App.cpp

```diff
@@ -25,14 +25,14 @@ using namespace Windows::ApplicationModel;
 App::App() noexcept
 {
 #if BUNDLE
-    // Note: add <UseBundle>true</UseBundle> property to application's ReactNativeWindowsProps PropertyGroup to force BUNDLE to true
     JavaScriptBundleFile(L"index.windows");
     InstanceSettings().UseWebDebugger(false);
     InstanceSettings().UseFastRefresh(false);
 #else
     JavaScriptMainModuleName(L"index");
-    InstanceSettings().UseWebDebugger(true);
+    InstanceSettings().UseWebDebugger(false); // BabylonReactNative accesses the jsi runtime, which isn't possible with the web debugger

```

Comment:
On other platforms, web debugger (remote debugging) is enabled/disabled dynamically from the app's debug/developer menu. Is this different on Windows? It would be annoying if all UWP RN apps have to change this configuration in order to use Babylon React Native.

Reply:
it sounds like it may be possible to set this dynamically, but i'm still trying to track down documentation on how. For the time being, this hardcoding of the ip address is a valid workaround/has been allowing me to test things on hololens 2